src/main/resources/ownership.json

```diff
@@ -94,6 +94,22 @@
       "Fork"
     ]
   },
+  {
+    "name": "Apache Maven",
+    "repositories": [
+      {
+        "origin": "github.com",
+        "path": "apache/maven*",
+        "branch": "*"
+      }
+    ],
+    "commitOptions": [
+      "PullRequest",
+      "Branch",
+      "ForkAndPullRequest",
+      "Fork"
```

Comment:
Wasn't sure here; if this is all the options might make sense to just drop this. I want to keep the option open for some of the Apache Maven folks to add commits directly themselves.
```suggestion
```

Reply:
Yeah, this will limit the options. So direct commit wouldn't be an option shown in the commit dialog in the saas.

Reply:
If you want to allow all available commit options then omit this property.